mod/starred.php

```diff
@@ -7,19 +7,23 @@ function starred_init(&$a) {
 
 	$starred = 0;
 
-	if(! local_user())
+	if (! local_user()) {
 		killme();
-	if($a->argc > 1)
+	}
+	if ($a->argc > 1) {
 		$message_id = intval($a->argv[1]);
-	if(! $message_id)
+	}
+	if (! $message_id) {
 		killme();
+	}
 
 	$r = q("SELECT starred FROM item WHERE uid = %d AND id = %d LIMIT 1",
 		intval(local_user()),
 		intval($message_id)
 	);
-	if(! dbm::is_result($r))
+	if (! dbm::is_result($r)) {
 		killme();
+	}
 
 	if(! intval($r[0]['starred']))
```

Comment:
Standards: Could you please add brackets to this conditional statement? 😊
Standards: Could you please add a space after the `if`? 😊